script/thrift_gen.sh

```diff
@@ -0,0 +1,27 @@
+#!/bin/sh
+
+set -e
+
+cd "$(dirname "$0")/.."
+
+# checkout jaeger thrift files
+rm -rf jaeger-idl
+git clone https://github.com/jaegertracing/jaeger-idl
+
+# define thrift cmd
+THRIFT="docker run -u $(id -u) -v '${PWD}:/data' jaegertracing/thrift:0.13 thrift -o /data/jaeger-idl"
+THRIFT_CMD="${THRIFT} --gen php:psr4,oop"
```

Comment:
The output from when I ran this locally was:

```
docker run -u 502 -v '/Users/bstrecansky/git/opentelemetry-php:/data' jaegertracing/thrift:0.13 thrift -o /data/jaeger-idl --gen php:psr4,oop /data/jaeger-idl/thrift/baggage.thrift
[WARNING:generation:1] psr4 is default option! needn't add psr4 option!
```

Perhaps you don't need to explicitly state the psr4 option?  Were you adding this for succinctness?

Reply:
Removed psr4 explicit check as it's already the default option.